subprojects/soak/src/integTest/groovy/org/gradle/vfs/VirtualFileSystemRetentionSoakTest.groovy

```diff
@@ -72,10 +73,11 @@ class VirtualFileSystemRetentionSoakTest extends DaemonIntegrationSpec implement
         NUMBER_OF_REPETITIONS.times { iteration ->
             changeAllSourceFiles(iteration)
             waitForChangesToBePickedUp()
-            succeeds("assemble", "--parallel")
+            succeeds("assemble")
```

Comment:
Why do we run with `--parallel`?

Reply:
So the test is faster on Windows.

Reply:
What's different about Windows?

Reply:
The test is faster on all OSes, though Windows is the slowest one where it makes a difference.